public/css/style.css

```diff
@@ -169,21 +184,179 @@ background: linear-gradient(rgba(96, 99, 100, 0.3), rgba(0,0,0,0.3));
   .play-foot, .login-btn {
     font: Bold 2rem 'Quicksand';
   }
-
-  #donations {
-    text-align: left;
+}
+.login-image-overlay {
+  width: 100%;
+  background:  linear-gradient(rgba(10, 10, 10, 0.6), rgba(10, 10, 10, 0.6)),
+      url('https://res.cloudinary.com/dqsmurjpg/image/upload/v1530770621/3-cards-against-humanity_32938.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
```

Comment:
Add spacing after your css selector
```
selector {
    property: value;
} // <---space here

another-selector {
    property: value;
}
```

Make the change to all affected parts of this file and to every css file affected.

Reply:
Done